docs/source/cli_configure.rst

```diff
@@ -10,12 +10,12 @@ The ``qpc server config`` command configures the connection between the qpc comm
 The ``qpc server config`` command takes the following options:
 
 - The ``--host`` option is required. If you are using the qpc command line interface on the same system where the server is running, you can supply the loopback address ``127.0.0.1`` as the value. Otherwise, supply the IP address for your Quipucords server.
-- The ``--port`` option is optional. The default value for this option is ``443``. If you decided to remap the Quipucords default exposed server port to another port, the port option is required. You must supply the port option and the remapped value in the command, for example, ``--port 8443``.
+- The ``--port`` option is optional. The default value for this option is ``9443``. If you decided to remap the Quipucords default exposed server port to another port, the port option is required. You must supply the port option and the remapped value in the command, for example, ``--port 9443``.
```

Comment:
The second part of this is talking about not using the default port (remapping the port), so give a different port number or leave it at ``--port 8443``.